lms/djangoapps/discussion/tasks.py

```diff
@@ -90,3 +93,26 @@ def _get_thread_url(context):
         'id': context['thread_id'],
     }
     return urljoin(settings.LMS_ROOT_URL, permalink(thread_content))
+
+
+def _generate_ga_pixel_url(course_key, user_id):
```

Comment:
I wonder if we should have a common base template that includes stuff like this in all emails that go out from the LMS?